.github/workflows/test.yml

```diff
@@ -0,0 +1,91 @@
+name: Timber starter theme tests
+
+on:
+  push:
+    branches:
+      - '1.x'
+      - '2.x'
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+
+# Cancel previous workflow run groups that have not completed.
+concurrency:
+  # Group workflow runs by workflow name, along with the head branch ref of the pull request
+  # or otherwise the branch or tag ref.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.4', '8.0', '8.1']
+        wp: ['latest']
+        multisite: ['0', '1']
+        dependency-version: ['highest']
+        extensions: ['gd']
+        experimental: [false]
+        include:
+          # WP Trunk
+          - php: '7.4'
+            wp: 'trunk'
+            dependency-version: 'highest'
+            multisite: '0'
+            experimental: true
+          # PHP 8.0
+          - php: '8.0'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
+            multisite: '0'
+            experimental: false
+          # PHP 8.1
+          - php: '8.1'
+            wp: 'latest'
+            dependency-version: 'prefer-stable'
```

Comment:
⚠️ There is not "prefer-stable" version.
Only lowest/locked/highest.

Using `prefer-stable/lowest` is a popular myth. Please see https://gist.github.com/szepeviktor/200b45f9805dc8bf3f185f0ade216987

Reply:
The way I see it, we could just remove `dependency-version` all together in the workflow and let Composer decide, right? For the starter theme, we only need to test against the latest versions, because what it’s going to b used with.

Reply:
> we only need to test against the latest versions

And we don't have other versions :)

Testing with lowest version **will** reveal problem in the long-term.
https://github.com/szepeviktor/byte-level-care/blob/a1cf6cc74a7b98deb28ff5b97feec9784a45be29/.github/workflows/back-end.yml#L84-L89
I see that every month 👁️

Reply:
Updated in [`244d315` (#142)](https://github.com/timber/starter-theme/pull/142/commits/244d31576f6c96d7db6b36272dad1164924243ea).